README.md

```diff
@@ -8,39 +8,34 @@ Nodejs with Express app to present the marketing pages.
 
   
 
-## Prerequisite
+## Requirements
+- Node 10
 
-  
-
-Install the long-term support (LTS) version of <a  href="https://nodejs.org/en/">Node.js</a>, which includes NPM.
+Install the long-term support (LTS) version of <a href="https://nodejs.org/en/">Node.js</a>, which includes NPM.
```

Comment:
I just thought. Is version 10 still in LTS?

Reply:
![image](https://user-images.githubusercontent.com/37105695/70814499-8414b800-1dc3-11ea-8b98-767cc1c8db1c.png)

It still should be

Reply:
Good 👍